src/spot.sol

```diff
@@ -37,22 +37,22 @@ contract Spotter is LibNote {
 
     // --- Data ---
     struct Ilk {
-        PipLike pip;
-        uint256 mat;
+        PipLike pip;  // OSM
+        uint256 mat;  // Liquidation ratio [ray]
     }
 
     mapping (bytes32 => Ilk) public ilks;
 
-    VatLike public vat;
-    uint256 public par; // ref per dai
+    VatLike public vat; // CDP Engine
+    uint256 public par; // ref per dai     [ray]
```

Comment:
nit: minimum of two spaces between code and start of comment

Also I'm not convinced it's aesthetically more pleasing to align the `[ray]` on this line with the one on line 41, as opposed to just giving it a natural single space.

Reply:
You're right - will give it a natural single space